src/devices/Mcp25xxx/Register/AcceptanceFilter/RxFilterNumber.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Mcp25xxx.Register.AcceptanceFilter
+{
+    /// <summary>
+    /// Receive Filter Number.
+    /// </summary>
+    public enum RxFilterNumber
```

Comment:
I'd just directly use int/byte wherever you use it and perhaps do some validation on that since the values match numbers

Reply:
Yeah, that's true.  Will update.